Actions/DetermineDeploymentEnvironments/DetermineDeploymentEnvironments.ps1

```diff
@@ -11,7 +11,7 @@ function IsGitHubPagesAvailable() {
     $url = "$($ENV:GITHUB_API_URL)/repos/$($ENV:GITHUB_REPOSITORY)/pages"
     try {
         Write-Host "Requesting GitHub Pages settings from GitHub"
-        $ghPages = InvokeWebRequest -Headers $headers -Uri $url -ignoreErrors | ConvertFrom-Json
+        $ghPages = (InvokeWebRequest -Headers $headers -Uri $url).Content | ConvertFrom-Json
```

Comment:
Why not make `InvokeWebRequest` return `$result.Content`?

Reply:
Because InvokeWebRequest is a enhanced version of Invoke-WebRequest - it should return anything different.
I could also have changed things to use Invoke-RestMethod, which returns the Json object directly, but I wanted this change to have as little impact/side effects as possible.